frontend/src/old-pages/Clusters/__tests__/Clusters.test.tsx

```diff
@@ -0,0 +1,121 @@
+import { ThemeProvider } from '@emotion/react'
+import { createTheme } from '@mui/material'
+import { render, waitFor, screen, prettyDOM } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { SnackbarProvider } from 'notistack'
+import { I18nextProvider } from 'react-i18next'
+import { QueryClient, QueryClientProvider } from 'react-query'
+import { Provider } from 'react-redux'
+import { BrowserRouter, useNavigate } from 'react-router-dom'
+import i18n from '../../../i18n'
+import { ListClusters } from '../../../model'
+import { store, isAdmin } from '../../../store'
+import Clusters from '../Clusters'
+
+
+const queryClient = new QueryClient();
+const mockClusters = [{
+  clusterName: 'test-cluster',
+  clusterStatus: 'CREATE_COMPLETE',
+  version: '3.1.4',
+  cloudformationStackArn: 'arn',
+  region: 'region',
+  cloudformationStackStatus: 'status'
+}];
+
+const MockProviders = (props: any) => (
+  <QueryClientProvider client={queryClient}>
+    <I18nextProvider i18n={i18n}>
+      <Provider store={store}>
+        <ThemeProvider theme={createTheme()}>
+          <SnackbarProvider>
+            <BrowserRouter>
+              {props.children}
+            </BrowserRouter>
+          </SnackbarProvider>
+        </ThemeProvider>
+      </Provider>
+    </I18nextProvider>
+  </QueryClientProvider>
+)
+
+jest.mock('../../../model', () => ({
+  ListClusters: jest.fn(),
+}));
+
+jest.mock('../../../store', () => ({
+  ...jest.requireActual('../../../store') as any,
+  isAdmin: () => true,
+}));
+
```

Comment:
suggestion: FYI, we should use a [mocked store](https://github.com/reduxjs/redux-mock-store). We can leave it for the next time